SemDiff.Core/GitHub.cs

```diff
@@ -207,7 +269,7 @@ public class PullRequest
 
             [JsonProperty("updated_at")]
             public DateTime Updated { get; set; }
-
+            public DateTime LastWrite { get; set; }
```

Comment:
That can be deleted from this pull request. I was working on 2 different things and decided to make a pull request for just deletion and thought I removed everything else.